bigtop-tests/smoke-tests/kafka/TestKafkaSmoke.groovy

```diff
@@ -32,20 +32,7 @@ class TestKafkaSmoke {
   static Shell sh = new Shell("/bin/bash -s");
 
   static final String KAFKA_HOME = "/usr/lib/kafka"
-  static final String KAFKA_CONFIG = KAFKA_HOME + "/config/server.properties "
   static final String KAFKA_TOPICS = KAFKA_HOME + "/bin/kafka-topics.sh "
-  static final String KAFKA_SERVER_START = KAFKA_HOME + "/bin/kafka-server-start.sh "
-  static final String KAFKA_SERVER_STOP = KAFKA_HOME + "/bin/kafka-server-stop.sh "
-
-  @BeforeClass
-  static void kafkaSetUp() {
```

Comment:
The idea in here is to run the smoke tests with kafka already up with the delete.topic.enable flag right?

Reply:
The original code is trying to change the configuration value of by restating Kafka broker. Just running `/bin/kafka-server-stop.sh` does not work if the Kafka broker was started as a service via systemctl. `--smoke-tests` of docker provisioner is the case.

Reply:
I agree, makes sense.